charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml

```
@@ -63,7 +63,7 @@ spec:
       initContainers:
       - name: copy-charts
         image: "{{ .Values.helmRepository.images.bundleInitializer.repository }}:{{ default $.Chart.AppVersion .Values.helmRepository.images.bundleInitializer.tag }}"
-        command: ["/bin/cp", "-r", "/charts/",  "/helm-charts/bundles/"]
+        command: ["/bin/cp", "-r", "/charts/.",  "/helm-charts/bundles/"]
```

I don't understand how this works but it does seem to! Do you have any docs on it?

---

I wonder if using `--no-target-directory` (`-T`) is better defined? From [https://unix.stackexchange.com/posts/94838/revisions this is equivalent](https://unix.stackexchange.com/a/94838) I guess, but perhaps more understandable than the difference between `/charts/` and `/charts/.`?

Regardless, let's add a comment here explaining why this command is specified as it is.

Optional suggestion:

```suggestion
        # Copy the charts bundled in the image to the mounted PVC.
        # The combination of flags `--recursive --no-target-directory` ensure that the bundled directory contents
        # only are copied to the destination, rather than copying the source directory itself to the destination,
        # which would lead to incorrect nesting. See https://unix.stackexchange.com/a/94838 for further
        # explanation.
        # Globs cannot be used here because we are not running in a shell and `cp` does not natively support
        # globbing.
        command: ["/bin/cp", "--recursive", "--no-target-directory", "--verbose", "/charts/",  "/helm-charts/bundles/"]
```

---

Thank you for the `--no-target-directory` and the comment! 
Tested locally too upgrading on a live cluster.